driver/vhci_ude/vhci_plugin.c

```diff
@@ -61,7 +61,7 @@ setup_with_dsc_conf(pctx_vusb_t vusb, PUSB_CONFIGURATION_DESCRIPTOR dsc_conf)
 	if (dsc_conf->bNumInterfaces > 0) {
 		int	i;
 
-		vusb->intf_altsettings = (PSHORT)ExAllocatePoolWithTag(PagedPool, dsc_conf->bNumInterfaces * sizeof(SHORT), VHCI_POOLTAG);
```

Comment:
Yes, when running under the verifier (can't remember exact settings).

Reply:
I'll check with a verifier.

Reply:
I have checked with a verifier. 
NonPagedPool is a scarce resource, so I want to use PagedPool allocation if possible.
There's 2 steps to resolve this issue.
Let WDF call EvtIoDeviceControl at a passive level. This can be done easily by changing callback attribute.
Do not call `plugin_vusb` with a lock held. Because a lock increases IRQL.

Reply:
Yes it is a scarce resource, but we are talking about (a few interfaces * sizeof(SHORT/*2*/) * MAX_NUM_PORTS) bytes at most?

p.s. a happy and healthy new year